Not working on Linux

The same issue that I've seen at ikvm-revived/ikvm#213 also seems to affect this package:
```
[user@laptop-host ikvmtest]$ dotnet build
Microsoft (R) Build Engine version 17.0.1+b177f8fa7 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
/home/user/.nuget/packages/ikvm.maven.sdk/1.1.0/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.targets(8,9): error MSB4216: Could not run the "MavenReferenceItemImport" task because MSBuild could not create or connect to a task host with runtime "CLR4" and architecture "x64".  Please ensure that (1) the requested runtime and/or architecture are available on the machine, and (2) that the required executable "/usr/lib64/dotnet/sdk/6.0.110/MSBuild.exe" exists and can be run. [/home/user/ikvmtest/ikvmtest.csproj]
/home/user/.nuget/packages/ikvm.maven.sdk/1.1.0/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.targets(9,13): error MSB4028: The "MavenReferenceItemImport" task's outputs could not be retrieved from the "Items" parameter. Object does not match target type. [/home/user/ikvmtest/ikvmtest.csproj]

Build FAILED.

/home/user/.nuget/packages/ikvm.maven.sdk/1.1.0/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.targets(8,9): error MSB4216: Could not run the "MavenReferenceItemImport" task because MSBuild could not create or connect to a task host with runtime "CLR4" and architecture "x64".  Please ensure that (1) the requested runtime and/or architecture are available on the machine, and (2) that the required executable "/usr/lib64/dotnet/sdk/6.0.110/MSBuild.exe" exists and can be run. [/home/user/ikvmtest/ikvmtest.csproj]
/home/user/.nuget/packages/ikvm.maven.sdk/1.1.0/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.targets(9,13): error MSB4028: The "MavenReferenceItemImport" task's outputs could not be retrieved from the "Items" parameter. Object does not match target type. [/home/user/ikvmtest/ikvmtest.csproj]
    0 Warning(s)
    2 Error(s)
```
Updated my test project to IKVM 8.3.3 and added IKVM.Maven.Sdk 1.1.0.

--- a/src/IKVM.Maven.Sdk/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.props
+++ b/src/IKVM.Maven.Sdk/buildTransitive/netstandard2.0/IKVM.Maven.Sdk.props
@@ -2,7 +2,7 @@
     <PropertyGroup>
         <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     </PropertyGroup>
-    
+
     <Import Project="$(MSBuildThisFileDirectory)..\..\targets\generated.props" Condition="Exists('$(MSBuildThisFileDirectory)..\..\targets\generated.props')" />
 
     <PropertyGroup>
@@ -12,14 +12,16 @@
     <PropertyGroup>
         <IkvmMavenSdkTaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netcoreapp3.1</IkvmMavenSdkTaskFolder>
         <IkvmMavenSdkTaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net472</IkvmMavenSdkTaskFolder>
+        <IkvmMavenSdkTaskFactory Condition=" '$(MSBuildRuntimeType)' == 'Core' "></IkvmMavenSdkTaskFactory>
+        <IkvmMavenSdkTaskFactory Condition=" '$(MSBuildRuntimeType)' != 'Core' ">TaskHostFactory</IkvmMavenSdkTaskFactory>
         <IkvmMavenSdkTaskAssembly>$(IkvmMavenSdkDir)tasks\$(IkvmMavenSdkTaskFolder)\IKVM.Maven.Sdk.Tasks.dll</IkvmMavenSdkTaskAssembly>
     </PropertyGroup>
 
-    <UsingTask TaskName="MavenReferenceItemImport" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="TaskHostFactory" />
-    <UsingTask TaskName="MavenReferenceItemPrepare" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="TaskHostFactory" />
-    <UsingTask TaskName="MavenReferenceItemResolve" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="TaskHostFactory" />
-    <UsingTask TaskName="MavenReferenceItemInstall" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="TaskHostFactory" />
-    <UsingTask TaskName="MavenWriteProjectObjectModelFile" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="TaskHostFactory" />
+    <UsingTask TaskName="MavenReferenceItemImport" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="$(IkvmMavenSdkTaskFactory)" />
+    <UsingTask TaskName="MavenReferenceItemPrepare" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="$(IkvmMavenSdkTaskFactory)" />
+    <UsingTask TaskName="MavenReferenceItemResolve" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="$(IkvmMavenSdkTaskFactory)" />
+    <UsingTask TaskName="MavenReferenceItemInstall" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="$(IkvmMavenSdkTaskFactory)" />
+    <UsingTask TaskName="MavenWriteProjectObjectModelFile" AssemblyFile="$(IkvmMavenSdkTaskAssembly)" TaskFactory="$(IkvmMavenSdkTaskFactory)" />
 
     <ItemDefinitionGroup>
         <MavenReference>

